OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/FullscreenLoaderDialogFragment.kt

```diff
@@ -182,4 +183,8 @@ class FullscreenLoaderDialogFragment : DialogFragment() {
         fun getRetainedInstance(fragmentManager: FragmentManager?): FullscreenLoaderDialogFragment? =
             fragmentManager?.findFragmentByTag(TAG) as FullscreenLoaderDialogFragment?
     }
+
+    interface OnCloseListener {
```

Comment:
IMO, better to rename it as `FullScreenDismissListener` and method -> `onDismiss()`

Reply:
done.